modules/etcd-overview.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="etcd-overview_{context}"]
+= Overview of etcd
+
+etcd is a consistent, distributed key-value store that holds small amounts of data that can fit entirely in memory. Although etcd is a core component of many projects, it is the primary data store for Kubernetes, which is the standard system for container orchestration.
+
+[id="etcd-benefits_{context}"]
+== Benefits of using etcd
+
+By using etcd, you can enjoy several benefits:
+
+* Maintain consistent uptime for your cloud-native applications, and keep them working even if individual servers fail
+* Store and replicate all cluster states for Kubernetes
+* Distribute configuration data to provide redundancy and resiliency for the configuration of nodes
+
+[id="etcd-architecture_{context}"]
+== How etcd works
+
+To ensure a reliable approach to cluster configuration and management, etcd uses the etcd Operator. The Operator simplifies the use of etcd on a Kubernetes container platform like {product-title}. With the etcd Operator, you can create or delete etcd members, resize clusters, perform backups, and upgrade etcd.
+
+The etcd Operator observes, analyzes, and acts:
+
+. It observes the cluster state by using the Kubernetes API.
+. It analyzes differences between the current state and the state that you want.
+. It fixes the differences through the etcd cluster management APIs, the Kubernetes API, or both.
+
+etcd holds the cluster state, which is constantly updated. This state is continuously persisted, what leads to a high number of small changes at high frequency. As a result, it is critical to back etcd cluster member with fast, low-latency I/O. For more information about best practices for etcd, see "Recommended etcd practices".
```

Comment:
s/what/which ??

s/back etcd cluster member/back **_the_** etcd cluster member ??

Reply:
Likely my ignorance: is _back_ here the correct word? This is different than back up?

Reply:
Good catches on the missing "the" and the "what" that should have been "which".

It turns out that "back" is the correct word 👍